openstack/compute/v2/flavors/requests.go

```diff
@@ -200,3 +200,14 @@ func IDFromName(client *gophercloud.ServiceClient, name string) (string, error)
 		return "", err
 	}
 }
+
+// ExtraSpecs requests all the extra-specs for the given flavor ID.
+func ListExtraSpecs(client *gophercloud.ServiceClient, id string) (r ListExtraSpecsResult) {
+	_, r.Err = client.Get(extraSpecsListURL(client, id), &r.Body, nil)
+	return
+}
+
+func GetExtraSpecs(client *gophercloud.ServiceClient, id string, key string) (r GetExtraSpecsResult) {
```

Comment:
Maybe change `id` here to `flavorID` so it isn't confused with a supposed ExtraSpecID.

Reply:
Oh, also, should be `GetExtraSpec` singular?

Reply:
Done.